src/object-number.scss

```
@@ -0,0 +1,66 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-number+(--success | --warning | --error)
+*/
+$block: #{$fd-namespace}-object-number;
+
+$color-states: (
+  "positive": ("color": var(--sapPositiveTextColor)),
+  "critical": ("color": var(--sapCriticalTextColor)),
+  "negative": ("color": var(--sapNegativeTextColor)),
+  "informative": ("color": var(--sapInformativeTextColor))
+);
+
+.#{$block} {
+  // LARGE DESIGN VARIABLES
+  $fd-object-number-font-size-large: 1.5rem;
+
+  @include fd-reset();
+
+  color: var(--sapNeutralTextColor);
+  max-width: 100%;
+  display: inline-block;
+
+  &__text,
```

I believe it would be appropriate to include the `text-shadow` change in cases of low-contrast as is done for the `Object Status` component in  #1571 

```suggestion
  display: inline-block;
  text-shadow: var(--sapContent_TextShadow);

  &__text,
```